sentry/sentry.conf.example.py

```diff
@@ -103,6 +103,60 @@ def get_internal_network():
 # AI model prices from various public APIs.
 SENTRY_AIR_GAP = False
 
+# As of 25.9.0 (September 2025 release), Sentry enforces tighter restrictions
+# of allowed IP addresses for outgoing requests. This is to prevent
+# accidentally leaking sensitive information to third parties.
+# By default, Sentry will not allow requests to private IP addresses.
+# You can override this by configuring the allowed IP addresses here.
+SENTRY_DISALLOWED_IPS: tuple[str, ...] = (
```

Comment:
Is this list maintained on sentry repository and once you remove an IP from here you can send a request to that removed IP?

`SENTRY_DISALLOWED_IPS` is a little misleading name IMHO.

Reply:
Maybe it's better for self-hosted to maintain a `SENTRY_OUTGOING_ALLOWED_IPS` list and remove the items from getsentry/sentry list?

Not that IANA is going to change this list :) It has better readability for users to maintain allowed IPs.